src/backend/electron/fileHelper.ts

```diff
@@ -0,0 +1,15 @@
+import fs from "fs";
+import { moveFile } from "move-file";
+
+export async function writeFileSafely(
+  path: string,
+  data: string | NodeJS.ArrayBufferView,
+) {
+  // ファイル書き込みに失敗したときに設定が消えないように、tempファイル書き込み後上書き移動する
```

Comment:
コメントはこう書いておくと、VSCodeで関数をマウスオーバーした時にドキュメントが表示されてちょっと便利だったりします！

/** ファイル書き込みに失敗したときに設定が消えないように、tempファイル書き込み後上書き移動する */
```suggestion
export async function writeFileSafely(
  path: string,
  data: string | NodeJS.ArrayBufferView,
) {
```